source/gui/__init__.py

```diff
@@ -530,6 +482,7 @@ def __init__(self, frame):
 			self.Bind(wx.EVT_MENU, lambda evt: os.startfile(getDocFilePath("contributors.txt", False)), item)
 			# Translators: The label for the menu item to open NVDA Welcome Dialog.
 			item = menu_help.Append(wx.ID_ANY, _("We&lcome dialog..."))
+			from .startupDialogs import WelcomeDialog

```

Comment:
I believe this can be moved to the top of the file

Reply:
Are you sure this would not reintroduce circular imports?